### PR TITLE
Add Catalan language

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@
 - Feature: [#4916] FreeBSD support.
 - Feature: [#4963] Add boosters (from RCT1 and RCTC).
 - Feature: [#5113] Entertainers are now hired with a random costume.
+- Feature: Added Catalan translation.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: #5222 ?

**Comment by J0anJosep** [comment]
Ok!
I would prefer RCT2_LANGUAGE_ID_ENGLISH_UK instead of the Spanish identifier though.
Spanish identifier is also correct.

**Comment by Gymnasiast** [comment]
Using RCT2_LANGUAGE_ID_SPANISH means that OpenRCT2 will show object descriptions in Spanish if a Catalan translation is missing. I figured that most Catalan speakers are better at Spanish than at English, and thus would prefer Spanish in this case.

**Comment by J0anJosep** [comment]
Yes. Catalan speakers are better at Spanish than English. The Spanish identifier is also correct.

I am afraid that a Spanish translation may be misleading though. Providing the English version in that case would make clear for the user that no Catalan string was available.

**Comment by Gymnasiast** [comment]
Yes, it is indeed clearer that no Catalan string is available. On the other hand: there are a _lot_ strings in data files, and having the game half in English might cause some Catalan speakers to switch the entire game to Spanish.

Of course, if you plan to translate those strings (a few thousand, we do have a list for them) anytime soon it would make the whole matter largely irrelevant anyway.

**Pull request merged**